packages/api/tests/lambdas/test-clean_pg_executions.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
```

Comment:
What if we rolled these tests into test-cleanExecutions? Would that make sense? Again thinking about ES going away so eventually we'll be left with:

test-cleanExecutions
test-clean_pg_executions

which isn't much of a distinction at that point.

Reply:
I believe this should be taken care of with some consolidation of tests